recipe/build.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/build.py b/build.py
+index 91238f7..8adc550 100644
+--- a/build.py
++++ b/build.py
+@@ -1272,7 +1272,7 @@ def cmd_build_wx(options, args):
+ 
+     # Build the wx message catalogs, but first check that there is a msgfmt
+     # command available
+-    if findCmd('msgfmt'):
++    if findCmd('msgfmt') and not isWindows:
```

Comment:
Ouch! Sorry you had to do that to be able to build it in AppVeyor.

Reply:
No worries, I actually enjoy figuring this kind of stuff out (eventually). :)